lms/djangoapps/grades/tests/test_transformer.py

```diff
@@ -352,6 +352,7 @@ def test_course_version_collected_in_split(self):
             blocks = self.build_course_with_problems()
         block_structure = get_course_blocks(self.student, blocks[u'course'].location, self.transformers)
         self.assertIsNotNone(block_structure.get_xblock_field(blocks[u'course'].location, u'course_version'))
+        self.assertIsNotNone(block_structure.get_xblock_field(blocks[u'problem'].location, u'course_version'))
```

Comment:
Can you instead update this line to make sure the `course_version` value on the problem is equal to the one from the course?